src/chocolatey/GetChocolatey.cs

```diff
@@ -222,6 +222,15 @@ public IEnumerable<T> List<T>()
             var runner = new GenericRunner();
             return runner.list<T>(configuration, _container, isConsole: false, parseArgs: null);
         }
+
+        public int Count()

```

Comment:
GetListCount sounds fine to. Count isn't clear on what it is doing, that's why I was hoping for more. It would be different if you were calling `GetChocolatey().Setup().List().Count